TLM/TLM/UI/LegacySubToolMixin.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace TrafficManager.UI {
+    using System;
+    using ColossalFramework.UI;
+    using TrafficManager.U;
+    using UnityEngine;
+
+    [Obsolete("Used in legacy GUI for migration to new subtool base clas")]
+    public class LegacySubToolMixin {
+        [Obsolete("Used in legacy GUI")]
+        internal static GUILayoutOption[] LegacyEmptyOptionsArray = new GUILayoutOption[0];
+
+        [Obsolete("Used in legacy GUI")]
+        protected void LegacyDragWindow(ref Rect window) {
```

Comment:
if the class is obsolete do we have to mark every member as obsolete too?

Reply:
Marking class obsolete will underline only class name usages in editor.
Marking methods obsolete will underline method usages too.
Its for visual convenience, the class will be deleted in the end.